packages/usa-input/src/usa-input.twig

```diff
@@ -1,20 +1,24 @@
-<label class="usa-label" for="input-type-text">Text input label</label>
-{% if state == "error" %} <span class="usa-error-message" id="input-error-message" role="alert">Helpful error message</span>{%- endif %}
-<input class="usa-input
-  {% if state == "focus" %} usa-focus{%- endif%}
-  {% if state == "error" %} usa-input--error{%- endif%}
-  {% if state == "success" %} usa-input--success{%- endif%}"
-  {% if state == "disabled" %} disabled{%- endif%}
-  {% if state == "aria-disabled" %} aria-disabled="true"{%- endif%}
-  id="input-type-text" name="input-type-text" type="text">
-<label class="usa-label" for="input-type-textarea">Text area label</label>
-{% if state == "error" %}
-  <span class="usa-error-message" id="textarea-error-message" role="alert">Helpful error message</span>
-{%- endif %}
-<textarea class="usa-textarea
-  {% if state == "focus" %} usa-focus{%- endif%}
-  {% if state == "error" %} usa-input--error{%- endif%}
-  {% if state == "success" %} usa-input--success{%- endif%}"
-  {% if state == "disabled" %} disabled{%- endif%}
-  {% if state == "aria-disabled" %} aria-disabled="true"{%- endif%}
-  id="input-type-textarea" name="input-type-textarea"></textarea>
+{% if state == "error" %}<div class="usa-form-group usa-form-group--error">{%- endif %}
+  <label class="usa-label{% if state == "error" %} usa-label--error{%- endif %}" for="input-type-text">Text input label</label>
+  {% if state == "error" %} <span class="usa-error-message" id="input-error-message" role="alert">Helpful error message</span>{%- endif %}
+  <input class="usa-input
+    {% if state == "focus" %} usa-focus{%- endif%}
+    {% if state == "error" %} usa-input--error{%- endif%}
+    {% if state == "success" %} usa-input--success{%- endif%}"
+    {% if state == "disabled" %} disabled{%- endif%}
+    {% if state == "aria-disabled" %} aria-disabled="true"{%- endif%}
+    id="input-type-text" name="input-type-text" type="text">
```

Comment:
**issue:** This is missing the `aria-describedby` attribute that ties it to the error message
```suggestion
    id="input-type-text" name="input-type-text" type="text"
    aria-describedby="input-error-message">
```

Reply:
Thanks for catching this! Resolved in [af4c99f](https://github.com/uswds/uswds/pull/5996/commits/af4c99f54c2b2eba2fa61748394e45eec8e81cd4)